content/_index.es.md

```diff
@@ -0,0 +1,136 @@
+Zig es un lenguaje de programación de propósito general y una cadena de herramientas para mantener software **robusto**, **optimo** y **reusable**.
+{{< /slogan >}}
+
+{{< flexrow class="container" style="padding: 20px 0; justify-content: space-between;" >}}
+{{% div class="features" %}}
+
+# ⚡ Un Lenguaje Simple
+Enfocate en depurar tu aplicación en vez de depurar tu conocimiento del lenguaje
+
+- Sin control de flujo oculto.
+- Sin asignaciones de memoria ocultas
+- Sin preprocesador, sin macros.
+
+# ⚡ Comptime
+Un enfoque fresco hacia la meta-programación basada en ejecución de codigo en tiempo de compilación y evaluación tardía (lazy evaluation). 
+
+- Llama cualquier función en tiempo de compilación.
+- Manipula tipos como si fuesen valores sin penalizar el tiempo de ejecución.
+- Comptime emula la arquitectura objetivo.
+
+# ⚡ Desempeño y Seguridad
+Escribe código claro y rapido capaz de manejar todas las condiciones de error.
+
+- Zig guía en forma eficaz tu lógica de tratamiento de errores.
+- Los chequeos configurables de tiempo de ejecución te ayudan a alcanzar un balance entre desempeño y garantías de seguridad.
+- Aprovecha las ventajas de los tipos vector para expresar instrucciones SIMD en forma portable.
+
+{{% flexrow style="justify-content:center;" %}}
+{{% div %}}
+<h1>
+    <a href="learn/overview/" class="button" style="display: inline;">Revisión a fondo</a>
+</h1>
+{{% /div %}}
+{{% div  style="margin-left: 5px;" %}}
+<h1>
+    <a href="learn/samples/" class="button" style="display: inline;">Más ejemplos de código</a>
+</h1>
+{{% /div %}}
+{{% /flexrow %}}
+{{% /div %}}
+{{< div class="codesample" >}}
+
+{{% zigdoctest "assets/zig-code/index.zig" %}}
+
+{{< /div >}}
+{{< /flexrow >}}
+
+
+{{% div class="alt-background" %}}
+{{% div class="container"  style="display:flex;flex-direction:column;justify-content:center;text-align:center; padding: 20px 0;" title="Comunidad" %}}
+
+{{< flexrow class="container" style="justify-content: center;" >}}
+{{% div style="width:25%" %}}
+<img src="/ziggy.svg" style="max-height: 200px">
+{{% /div %}}
+
+{{% div class="community-message" %}}
+# La comunidad Zig es descentralizada
+Todos son libres para iniciar y mantener su propio espacio de reunion para la comunidad.
+No existe el concepto de "oficial" o "no oficial", no obstante, cada lugar de reunion tiene sus propias reglas y moderadores.
+
+<div style="">
+<h1>
+	<a href="https://github.com/ziglang/zig/wiki/Community" class="button" style="display: inline;">Ver todas las comunidades</a>
+</h1>
+</div>
+{{% /div %}}
+{{< /flexrow >}}
+<div style="height: 50px;"></div>
+
+{{< flexrow class="container" style="justify-content: center;" >}}
+{{% div class="main-development-message" %}}
+# Desarrollo principal
+El repositorio de Zig se encuentra en [https://github.com/ziglang/zig](https://github.com/ziglang/zig), donde también hospedamos el seguimiento a problemas y discuciones de propuestas.
```

Comment:
nit: `discusiones`

Reply:
"El seguimiento a problemas" sounds a little strange to me grammatically, as if it's referring to the purpose of the site/tool, rather than the tool itself

Seems like it perhaps should be "herramienta de seguimiento de errores", "rastreador de errores", or similar

Reply:
I like your idea